CHANGELOG.md

```diff
@@ -81,6 +81,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - **CUMULUS-2792**
     - Remove API Granule model and all related test code
     - Remove granule-csv endpoint
+  - **CUMULUS-2856**
+    - Update API/Message write logic to handle nulls as deletion in execution PUT/message write logic
+    - Update execution PUT endpoint to no longer respect message write constraints and update all values passed in
```

Comment:
This might be a breaking change

Reply:
Resolved in https://github.com/nasa/cumulus/pull/3186/commits/05e4e6bec64eec060eeb397702228c8613031cb9